### PR TITLE
{math}[GCCcore/11.3.0] Givaro v4.2.0

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/g/Givaro/Givaro-4.2.0-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    ('GMP', '6.2.1'),
+]
+
+preconfigopts = "env NOCONFIGURE=1 ./autogen.sh && "
+configopts = "--with-gmp=$EBROOTGMP --enable-inline"
+
+sanity_check_paths = {
+    'files': ['bin/givaro-config', 'include/givaro-config.h'],
+    'dirs': ['bin', 'include', 'lib'],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/g/Givaro/Givaro-4.2.0-GCCcore-11.3.0.eb:39]:
```suggestion
}

sanity_check_commands = ["givaro-config --help"]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3120.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/176d2799143bdc5b9ec8d5f33ddd7cc9 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**